content/en/docs/guides/user/pipeline/triggers/jenkins/_index.md

```diff
@@ -29,3 +29,51 @@ description: >
     files](/docs/guides/user/pipeline/expressions/#property-files) section of the
     Pipeline Expression Guide for more information about how to specify and use
     property files.
+
+## Build Artifacts
+Jenkins has the ability to save files from a build as a build artifacts. These objects could be compiled binaries or kubernetes manifests or any other outputs created as part of the build process.
```

Comment:
s/artifacts/artifact

Reply:
"These could be binaries, Kubernetes manifests, or any other artifact created during the build process."

Reply:
Thanks @mattgogerly - I think I have got it right now :)